plenum/test/primary_selection/test_promotion_leads_to_primary_inconsistency.py

```diff
@@ -13,13 +13,15 @@
 nodeCount = 7
 
 
+whitelist = ["Node is not a validator"]
```

Comment:
Why didn't we require this whitelist before?

Reply:
it will be removed